oak-store-document/pom.xml

```diff
@@ -349,5 +349,11 @@
       <version>${testcontainers.version}</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
```

Comment:
(indentation)

Reply:
fixed in https://github.com/apache/jackrabbit-oak/pull/1459